### PR TITLE
fix : split  chars

### PR DESCRIPTION
**Pull request by mangotree2**
when prefix == stripPrefix ;
1.vkey = ""
2. keys := strings.Split(vkey, "/");
3. keys =  []string{""}
4.kvals[k] ;k =""
5.kval = map[""]interface{}

### CODE REVIEW

**Comment by mangotree2** [comment]
Thank you for guidance

File: source/etcd/util.go [hunk 1]
```diff
@@ -39,14 +39,27 @@ func update(e encoder.Encoder, data map[string]interface{}, v *mvccpb.KeyValue,
...
 
 	var vals interface{}
 	e.Decode(v.Value, &vals)
 
+	if !haveSplit && len(keys) == 1 {
+		switch action {
+		case "delete":
+			data = make(map[string]interface{})
+		default:
+			_v,ok := vals.(map[string]interface{})
```
Comment (asim) [inline source/etcd/util.go]: Why is this defined as _v? This is very strange. Just put `v, ok`

**Comment by asim** [comment]
Thanks

**Comment by mangotree2** [comment]
Thank you very much for your careful guidance.

**Pull request merged**